packages/foam-vscode/static/dataviz/graph.js

```diff
@@ -344,8 +396,10 @@ try {
     switch (message.type) {
       case 'didUpdateGraphData':
         const graphData = augmentGraphInfo(message.payload);
-        console.log('didUpdateGraphData', graphData);
+        model.graphData = graphData;
         Actions.refresh(graphData);
+        console.log('didUpdateGraphData', graphData, model.graphData);
+        Actions.evaluate();
```

Comment:
the fact that here we call `refresh`, then `evaluate`, which in turn calls `refresh` again, feels odd - am I missing something?

Reply:
No, it was redundant. I simplified things now.